prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Nicholas Prodehl';
+    console.log(fullName);
+    var numberOfPets = '2';
+    console.log(numberOfPets);
+    var likesVegetables = 'true';
```

Comment:
Hi Nicholas, you'll want to leave the quotes off of these so that the JavaScript engine doesn't think you mean strings instead of the number `2` and the boolean `true`.

Reply:
Just updated my commit. Thanks for the input!